spec/Gaufrette/Adapter/FtpSpec.php

```diff
@@ -1,229 +0,0 @@
-<?php
-
-namespace spec\Gaufrette\Adapter;
-
-//hack - mock php built-in functions
-require_once 'functions.php';
-
-use Gaufrette\Filesystem;
-use PhpSpec\ObjectBehavior;
-
-class FtpSpec extends ObjectBehavior
```

Comment:
Sadly, it's not possible to make a spec for PHP8+ because of the connection object which is impossible to mock.